sql/moz-fx-data-shared-prod/telemetry_derived/desktop_engagement_clients_v1/backfill.yaml

```diff
@@ -0,0 +1,7 @@
+2024-05-30:
+  start_date: 2019-01-01
```

Comment:
Do we actually need it as far back as 2019?

Reply:
You're right, I checked, Vanessa said we only needed it to start of 2021 so I updated it - good catch!